analysis/table1.R

```diff
@@ -0,0 +1,216 @@
+# Load libraries ---------------------------------------------------------------
+print("Load libraries")
+
+library(magrittr)
+library(here)
+library(dplyr)
+
+
+# Specify redaction threshold --------------------------------------------------
+print("Specify redaction threshold")
+
+threshold <- 6
+
+# Source common functions ------------------------------------------------------
+print("Source common functions")
+
+source("analysis/utility.R")
+
+# Specify arguments ------------------------------------------------------------
+print("Specify arguments")
+
+args <- commandArgs(trailingOnly = TRUE)
+
```

Comment:
Replace lines 24 to 36 with:
```suggestion
print(length(args))
if (length(args) == 0) {
  cohort <- "vax"
  age_str <- "18;40;65;85"
} else {
  cohort <- args[[1]]
  age_str <- args[[2]]
}
```
(I am unclear where this weird command args code is coming from, it cropped up in PR https://github.com/opensafely/post-covid-respiratory/pull/80 too?!

Reply:
Very likely I saw someone else using the `commandArgs` function when processing inputs so I just copied it wholesale - although I think it already existed in `dataset_clean.R` (it was when I was adding the describe flag).

For your edit - it means the function will likely break in someway if only 1 argument is specified I think (it'll try to assign `args[[2]]` which wont exist - I don't know how R deals with that). If that's what we want I'll make the change.

Reply:
This bit of code is just for our convenience when we are working on development. It it just asking 'Am I receiving information from the command line? If not, then use these default values.' We put it in the if loop because otherwise you have to comment/uncomment all the time and remember to not push it to main, which was a faff. If you run the script from the command line without two arguments, it errors out, which is the behaviour we want.